lib/rubocop/cop/lint/env_fetch_string_default.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Lint
+      # All values in ENV are strings (or objects responding to #to_str).
+      # Always use strings as default values for ENV.fetch
```

Comment:
This can be a bit more detailed. Something like this, feel free to tweak:
```suggestion
      # Always use a string or string-like object (responds to `#to_str`) as the default value for `ENV.fetch`.
      # When the environment variable is present, the return value will always be a string. For consistency, the the default
      # value should also be a string.
      #
      # This cop only registers an offense when the value is known to not be a string and allows method
      # calls, constants, and similar.
```